### DISCUSSION TITLE
How to change the default YAML file indent?

### DISCUSSION THREAD

**Discussion by nukemiko** [redhat-developer/vscode-yaml#752] [Q&A] [answered]
I can't find settings similar to "default indentation" in YAML extension setting and `settings.json`. It seems to have a default value "2" that cannot be permanently modified, but I want to change it to 4. How can I change the default indentation?

**Comment by gorkem** [comment, marked as answer]
Is `editor.tabSize` what you are looking for as described on the [README](https://github.com/redhat-developer/vscode-yaml#extension-settings)  you can use those setting on the YAML section to give language-specific settings.
